pkg/cmd/server/serve.go

```diff
@@ -46,6 +47,12 @@ var ServeCmd = &cobra.Command{
 	GroupID: util.SERVER_GROUP,
 	Args:    cobra.NoArgs,
 	Run: func(cmd *cobra.Command, args []string) {
+		user := os.Getenv("USER")
```

Comment:
`if os.Getenv("USER") == "root"` ? :D

Reply:
Lol, yeah. Had a different idea here but decided not to go for it and the switch remained 😅